x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.test.tsx

```diff
@@ -50,47 +48,39 @@ describe('ResultHeader', () => {
 
   describe('score', () => {
     it('renders score if showScore is true ', () => {
-      const wrapper = shallow(
-        <ResultHeader showScore resultMeta={resultMeta} isMetaEngine={false} />
-      );
+      const wrapper = shallow(<ResultHeader {...props} showScore />);
       expect(wrapper.find('[data-test-subj="ResultScore"]').prop('value')).toEqual(100);
     });
 
     it('does not render score if showScore is false', () => {
-      const wrapper = shallow(
-        <ResultHeader showScore={false} resultMeta={resultMeta} isMetaEngine={false} />
-      );
+      const wrapper = shallow(<ResultHeader {...props} showScore={false} />);
       expect(wrapper.find('[data-test-subj="ResultScore"]').exists()).toBe(false);
     });
   });
 
   describe('engine', () => {
     it('renders engine name if this is a meta engine', () => {
-      const wrapper = shallow(
-        <ResultHeader
-          showScore
-          resultMeta={{
-            ...resultMeta,
-            id: '1',
-          }}
-          isMetaEngine
-        />
-      );
+      const wrapper = shallow(<ResultHeader {...props} isMetaEngine />);
       expect(wrapper.find('[data-test-subj="ResultEngine"]').prop('value')).toBe('my-engine');
     });
 
     it('does not render an engine if this is not a meta engine', () => {
-      const wrapper = shallow(
-        <ResultHeader
-          showScore
-          resultMeta={{
-            ...resultMeta,
-            id: '1',
-          }}
-          isMetaEngine={false}
-        />
-      );
+      const wrapper = shallow(<ResultHeader {...props} isMetaEngine={false} />);
       expect(wrapper.find('[data-test-subj="ResultEngine"]').exists()).toBe(false);
     });
   });
+
+  describe('actions', () => {
+    const actions = [{ title: 'View document', onClick: () => {}, iconType: 'eye' }];
+
+    it('renders ResultActions if actions have been passed', () => {
+      const wrapper = shallow(<ResultHeader {...props} actions={actions} />);
+      expect(wrapper.find(ResultActions).exists()).toBe(true);
+    });
+
+    it('does not render an', () => {
```

Comment:
'does not render result actions?'

Reply:
lmfao whoops :dead_inside:

```suggestion
    it('does not render ResultActions if no actions are passed', () => {
```